collections/number/add-commas-to-number.md

```diff
@@ -0,0 +1,5 @@
+~~~ javascript
+const addCommas = (num) => (!num) ? null : num.toString().split('.').map((n,i) => (i === 0) ? n.replace(/\B(?=(\d{3})+(?!\d))/g, ',') : n).join('.');
```

Comment:
Can you remove the check `!num`? All of snippets assume that users pass valid parameters. So it looks like
```js
const addCommas = (num) => [...`${num}`].map(...)
```

Reply:
Okay.

Reply:
Is it good now?